### PR TITLE
Releases/0.x

### PR DESCRIPTION
**Pull request by jessehouwing**
Fixes vulnerability in underscore in 0.x versions.

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/azure-pipelines-tool-lib/108)

### CODE REVIEW

**Review by DaniilShmelev — APPROVED** [review]
Tested these changes manually, found no issues

File: package-lock.json [hunk 1]
```diff
@@ -295,9 +317,9 @@
       "dev": true
     },
     "lodash": {
-      "version": "4.17.15",
-      "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.15.tgz",
-      "integrity": "sha512-8xOcRHvCjnocdS5cpwXQXVzmmh5e5+saE2QGoeQmbKmRS6J3VQppPOIt0MnmE+4xlZoumy0GPG0D0MVIQbNA1A==",
+      "version": "4.17.21",
```
[the remark this answers is not in the document]
Comment (jessehouwing) [reply package-lock.json:320]: Resulted in one more updated package. But not the one you're seeing...

[the remark this answers is not in the document]
Comment (anatolybolshakov) [reply package-lock.json:404]: @jessehouwing could you please share which node version do you use?

[the remark this answers is not in the document]
Comment (jessehouwing) [reply package-lock.json:404]: 13.3

[the remark this answers is not in the document]
Comment (jessehouwing) [reply package-lock.json:404]: I pushed a regenerated version of the lock file below. I trashed the `node_modules` folder and did a fresh pm install. I'm on node 13.3 on windows

**Review by anatolybolshakov — APPROVED** [review]
LGTM, thanks!

**Pull request merged**